.github/workflows/ci.yaml

```diff
@@ -62,6 +62,10 @@ jobs:
   integration:
     runs-on: ubuntu-latest
 
+    concurrency:
```

Comment:
Drive-by change to cancel previous integration test run for this PR if a dev pushes a bunch of commits in quick succession.